source/fe/fe_bernardi_raugel.cc

```diff
@@ -177,7 +177,6 @@ FE_BernardiRaugel<dim>::initialize_support_points()
     }
 }
 
-template class FE_BernardiRaugel<1>;
```

Comment:
What happens?

Reply:
It's the same. We unconditionally access array element 1. AFAICT, it doesn't really make sense to use this finite element for 1D anyway.

Reply:
I guess that makes sense. @GrahamBenHarper -- FYI.

Reply:
That's fine. There is no 1D Bernardi-Raugel element. :)